depthai_examples/include/common.hpp

```diff
@@ -0,0 +1,73 @@
+#ifndef COMMON_HPP_
+#define COMMON_HPP_
+
+#include <depthai_bridge/BridgePublisher.hpp>
+#include <depthai_bridge/DisparityConverter.hpp>
+#include <depthai_bridge/ImageConverter.hpp>
+#include <depthai_bridge/ImuConverter.hpp>
+#include <memory>
+
+#include "depthai/depthai.hpp"
+
+#ifndef IS_ROS2
+    #include <iostream>
+    #include <string>
+
+    #include "ros/ros.h"
+template <typename T>
+static void getParamWithWarning(ros::NodeHandle& pnh, const char* key, T val) {
+    bool gotParam = pnh.getParam(key, val);
+    if(!gotParam) {
+        std::stringstream ss;
+        ss << val;
+        ROS_WARN("Could not find param '%s' on node '%s'. Defaulting to '%s'", key, pnh.getNamespace().c_str(), ss.str().c_str());
+    }
+}
+#endif
+
+class PostProcessing {
+   public:
+    dai::MedianFilter getMedianFilter();
+
+    using TemporalMode = dai::RawStereoDepthConfig::PostProcessing::TemporalFilter::PersistencyMode;
+    TemporalMode getTemporalMode();
+
+    using DecimationMode = dai::RawStereoDepthConfig::PostProcessing::DecimationFilter::DecimationMode;
+    DecimationMode getDecimationMode();
+
+    void setMedianFilter(std::shared_ptr<dai::node::StereoDepth> stereo);
+
+    void setFilters(std::shared_ptr<dai::node::StereoDepth> stereo);
+
```

Comment:
This looks good for the init time settings. 
But to modify these at the run time create a function to share the pointer of the device to this class using `setDevice` for example after the device is initialized. And when setDecimationFilter(...) is called from a service or dynamic reconfigure it modifes it.  

Thoughts ?

Reply:
Same applies for Exposure setting.